opennms-doc/guide-admin/src/asciidoc/text/config-files/opennms.properties.adoc

```diff
@@ -0,0 +1,920 @@
+
+[[file-opennms-properties]]
+=== File: opennms.properties
+
+This properties file allows you to set global properties for the opennms application.
+These properties will be set as system properties at the beginning of Java startup.
+Any properties set on the Java command-line with `-Dproperty=value` will be overridden by these values.
+
+[options="autowidth"]
+|===
+| Type             | Java property file with key value pairs
+| Location         | `${OPENNMS_HOME}/etc`
+| Restart required | yes
+|===
+
+==== General
+
+[options="header, autowidth"]
+|===
+| Property | `opennms.bin.dir`
```

Comment:
Is it also possible to left-justify the property names so that they all line up vertically on the page? I don't like the centering since it makes it difficult to scan for property names.